masterlist.yaml

```diff
@@ -228,11 +233,16 @@ common:
   - &reqBetterMales
     <<: *reqModX
     subs: [ '[Better males - Beautiful nudes and faces - New hairstyles](http://www.nexusmods.com/skyrim/mods/2488)' ]
-
-  - &reqSkyUI # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
+  # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
+  - &reqSkyUI
     type: warn
     content: 'SkyUI.esp is not active in your load order. [SkyUI](http://www.nexusmods.com/skyrim/mods/3863) is required for this mod to be fully functional.'
     condition: 'not active("SkyUI.esp")'
+  - &reqRequiemPatch
+    type: warn
+    content:
+      - lang: en
+        text: 'Incompatible with: Requiem. You must install a compatibility patch.'
```

Comment:
Is there a site / page that hosts information on what the latest patches are, etc. that you could link to instead of specific patches?